modules/ding_ting_frontend/ding_ting_frontend.features.field_instance.inc

```diff
@@ -1175,7 +1179,7 @@ function ding_ting_frontend_field_default_field_instances() {
           'prefix_type' => 'no',
         ),
         'type' => 'ting_title_default',
-        'weight' => 4,
+        'weight' => 3,
       ),
       'search_result' => array(
         'label' => 'hidden',
```

Comment:
There are a lot of unrelated weight chances, but this is how features exports the config after I changed, so I think it's best to keep it like this?